css/style.css

```diff
@@ -1,6 +1,879 @@
+  align-items: center;
+  flex-wrap: nowrap;
+}
+
+.right-menu__link {
+  font-family: 'GothaPro', sans-serif;
+  font-size: 12px;
+  text-align: justify;
+  text-transform: uppercase;
+  font-weight: bold;
+  color: #000;
+  height: 13px;
+}
+
+.image {
+  padding: 0 0 0 4px;
+  vertical-align: -4px;
+}
+
+/*1 секция*/
+
+.imageblock {
+  background: url(../image/rob-potter-bg.svg);
+  height:281px;
+  margin: 0 0 50px 0;
+  background-size: cover;
+}
+
+.imageblock__title {
+  font-family: 'GothaPro', sans-serif;
+  font-size: 30px;
+  line-height: 29px;
+  text-align: justify;
+  text-transform: uppercase;
+  color: #fff;
+  width: 186px;
+  margin: 0 0 13px 0;
+  padding: 89px 0 0 60px;
+}
+
+.imageblock__link {
+  padding: 0 0 2px 60px;
+}
+
+.link-arrow__white {
+  font-family: 'GothaPro', sans-serif;
+  font-size: 12px;
+  line-height: 11px;
+  text-align: justify;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.arrow-white__img {
+  padding-left: 2px;
+}
+
+.arrow-white__scrimg,
+.arrow-white__tabimg {
+  display: none;
+}
+
+.shop__title {
+  font-family: 'GothaPro', sans-serif;
+  font-size: 22px;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000;
+  margin: 0 0 12px 0;
+}
+
+.shop-wrap {
+  display: flex;
+  justify-content: space-between;
+  height: 240px;
+  margin-bottom: 54px;
+}
+
+.sneaker-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 235px;
+}
+
+.sneaker-item::after {
+  content:"";
+  position: absolute;
+  background: #000;
+  width: 2px;
+  height: 100px;
+  opacity: .2;
+  margin: 49px 0 0 249px;
+}
+
+.sneaker-item:nth-child(4)::after {
+  display: none;
+}
+
+.sneaker-item__block {
+  align-self: center;
+  width: 185px;
+  margin-bottom: 6px;
+}
+
+.sneaker-item__img {
+  width: 100%;
+  height: 100%;
+}
+
+.sneaker-item__name {
+  font-family: 'GothaPro', sans-serif;
+  font-size: 12px;
+  line-height: 11px;
+  text-align: justify;
+  color: #000;
+  padding: 0 0 0 12px;
+  margin: 14px 0 7px 0;
+}
+
+.sneaker-item__wrap {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.sneaker-item__price {
+  font-family: 'GothaPro', sans-serif;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 13px;
+  text-align: justify;
+  color: #000;
+  margin: 1px 0 0 12px;
+}
+
+.colour-circle {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 53px;
+  align-items: center;
+  padding: 0 17px 0 0;
+}
+
+.colour-circle__item {
+  width: 7px;
+  height: 7px;
+  border-radius: 50%;
+}
+
+.colour-circle__item_active {
+  width: 9px;
+  height: 9px;
+  border-radius: 50%;
+}
+
+.colour-circle__item_grey {
+  background: #C4C4C4;
+}
+
+.colour-circle__item_black {
+  background: #0E1211;
+}
+
+.colour-circle__item_red {
+  background: #B62E3C;
+}
+
+.colour-circle__item_yellow {
+  background: #FDB431;
+}
+
+.colour-circle__item_orange {
+  background: #E29D69;
```

Comment:
Codacy found an issue: [Expected "#E29D69" to be "#e29d69" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5730635)